leo/core/leoCommands.py

```diff
@@ -3593,6 +3594,191 @@ def looksLikeDerivedFile(self, fn: str) -> bool:
         except Exception:
             g.es_exception()
             return False
+    #@+node:ekr.20250717132026.1: *4* c.makeLinkLeoFiles & helper
+    def makeLinkLeoFiles(self,
+        *,
+        extensions: list[str],  # List of file extensions for generated @<file> nodes.
+        top_directory: str,
+        kind: str = '@clean',  # Any @<file> type. @clean is recommended.
+        report_changed_at_clean_nodes: bool = True,  # Recommended.
+        sub_directories: list[str] = None,
+        sub_outline_name: str = None,
```

Comment:
that sub_outline_name parameter is totally unused. It is overwritten in the `if files:` just before its usage. And then, all subsequent reference to it use that newly created value. 

So the original sub_outline_name  parameter passed to that method is unused completely.